benefits/enrollment_switchio/views.py

```diff
@@ -0,0 +1,30 @@
+from django.http import JsonResponse
+from django.views.generic import TemplateView, View
+
+from benefits.core import models, session
+
+
+class IndexView(TemplateView):
+    """CBV for the enrollment landing page."""
+
+    template_name = "enrollment_switchio/index--switchio.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        flow = session.flow(self.request)
+        context.update(
+            {
+                **flow.enrollment_index_context,
+                "cta_button": "tokenize_card",
+                "enrollment_method": models.EnrollmentMethods.DIGITAL,
+            }
+        )
+        return context
+
+
+class GatewayUrlView(View):
+    """CBV for the tokenization gateway registration"""
```

Comment:
Same comment as above - no need to document that this is a class-based view. Suggestion is to say "View handler" or "View" in place of "CBV"